pkg/model/manifest_test.go

```diff
@@ -1147,6 +1148,53 @@ func Test_GetTimeout(t *testing.T) {
 	}
 }
 
+func Test_ExpandVarsInSection(t *testing.T) {
+	dependency := Dependency{
+		Repository:   "${REPO}",
+		Branch:       "${NOBRANCHSET-$BRANCH}",
+		ManifestPath: "${NOMPATHSET=$MPATH}",
+		Namespace:    "${FOO+$NAMESPACE}",
+		Variables: Environment{
+			EnvVar{
+				Name:  "MYVAR",
+				Value: "${AVARVALUE}",
+			},
+			EnvVar{
+				Name:  "$${ANAME}",
+				Value: "MyValue",
+			},
+		},
+	}
+	expected := Dependency{
+		Repository:   "my/repo",
```

Comment:
where are the envs for the tests set? 🤔  are they global?

Reply:
can we set some env value to multiline to check this behavour?

Reply:
this function doesn't use environment variables. It just uses variables coming from `deploy variables` retrieved from configmap. The variables are stored in config map by the backend